config.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"net/url"
+
+	kingpin "gopkg.in/alecthomas/kingpin.v2"
+)
+
+type MateConfig struct {
+	Producer string
+	Consumer string
+	Debug    bool
+	SyncOnly bool
+
+	FakeDNSName       string
+	FakeMode          string
+	FakeTargetDomain  string
+	FakeFixedDNSName  string
+	FakeFixedIP       string
+	FakeFixedHostname string
+
+	KubeServer     *url.URL
+	KubeFormat     string
+	TrackNodePorts bool
```

Comment:
how about, for consistency, with the flags to prefix with `kubernetes` or change the flags to `kube`? (the latter would require a version bump)